### ISSUE TITLE
Sidebar not showing - it might be too early to use the "dvh" unit

### ISSUE THREAD

**Issue by outloudvi** [syuilo/Misskey#9636]
tl;dr: Upgrade to **Chrome for Android 108+/Firefox for Android 107+/Safari 15.4+** if you cannot call out the sidebar.

## 💡 Summary

The sidebar won't show after clicking that button.

Misskey 13 uses the new distance unit `dvh`:

<https://github.com/misskey-dev/misskey/blob/f8d09020807b6db6341bed43b1bb25a182ee29ca/packages/frontend/src/ui/_common_/navbar.vue#L116>

However, [it might not be widely supported enough](https://caniuse.com/viewport-unit-variants). Chrome for Android is released in Dec 2, 2022, and Edge 108 is released in Dec 5, 2022 - these versions are a bit too new.

I would suggest a compatibility fallback of `vh`:

``` css
> .body {
  height: 100vh;
  height: 100dvh; /* New browsers will apply this instead and old browsers will discard this */
}
```

## 🥰 Expected Behavior

The sidebar shows.

## 🤬 Actual Behavior

The sidebar doesn't show.

## 📝 Steps to Reproduce

1. Click the menu button.

## 📌 Environment

Misskey version: 13.0.0
Your OS: Android
Your browser: Chrome 102.

**Comment by outloudvi** [comment]
Duplicate of #9187.
